include/alpaka/queue/QueueCpuBlocking.hpp

```diff
@@ -217,3 +232,5 @@ namespace alpaka
         }
     }
 }
+
+#include <alpaka/event/EventCpu.hpp>
```

Comment:
Why is this strange include necessary?

Reply:
This is needed to pass the header checks where we check that a header is including all dependencies e.g. https://github.com/ComputationalRadiationPhysics/alpaka/pull/822/files#diff-5b4348d85ae1c4f4c1c432f2aa3b97bdR74 will call wait with an event but the specialization for the meaning of wait is within `EventCpu.hpp`. But since event depends on queue and here we depend of knowing the event we need to pull in the specializations.